src/vim9instr.c

```diff
@@ -1777,6 +1777,7 @@ generate_BLOBAPPEND(cctx_T *cctx)
     int
 generate_CALL(
 	cctx_T	    *cctx,
+	type_T	    *mtype,	// method type
```

Comment:
As the main argument for generate_CALL() is "ufunc", can you move the "mtype" argument to after "mi"?

Reply:
Done.